data/tlg0062/tlg001/tlg0062.tlg001.perseus-eng1.xml

```diff
@@ -0,0 +1,683 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">Phalaris</title>
+<author>Lucian of Samosata</author>
+<editor>Austin Morris Harmon, 1878-1950</editor>
+<funder>National Endowment for the Humanities</funder>
+<principal>Gregory Crane</principal>
+<respStmt>
+<persName>Gregory Crane</persName>
+<resp>OCRd, tagged and added initial corrections to  the text</resp>
+</respStmt>
+</titleStmt>
+<publicationStmt>
+<authority>Perseus Digital Library</authority>
+<idno type="filename">tlg0062.tlg001.1st1K-eng1.xml</idno>
+<availability>
+<licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence>
+</availability>
+<date>2023</date>
+<publisher>Trustees of Tufts University</publisher>
+<pubPlace>United States</pubPlace>
+</publicationStmt>
+<sourceDesc>
+<listBibl>
+<biblStruct>
+<monogr>
+<title>Lucian</title>
+<editor>
+<persName>
+<name>Austin Morris Harmon</name>
+</persName>
+</editor>
+<author ref="urn:cts:greekLit:tlg0062">Lucian of Samosata</author>
+<imprint>
+<publisher>Trustees of Tufts University</publisher>
+<pubPlace>Cambridge, MA</pubPlace>
+<date>1913</date>
+</imprint>
+<biblScope unit="volume">1</biblScope>
+</monogr>
+<ref target="https://hdl.handle.net/2027/uc1.b000604153">The Hathi Trust</ref>
+</biblStruct>
+</listBibl>
+</sourceDesc>
+</fileDesc>
+<encodingDesc>
+<p>Text encoded in accordance with the latest EpiDoc standards</p>
+<refsDecl n="CTS">
+    <cRefPattern matchPattern="(\w+).(\w+)"
+     replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:div[@n='$2'])">
+     <p>This pointer pattern extracts book and section</p>
+    </cRefPattern>
+    <cRefPattern matchPattern="(\w+)"
+     replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])">
+     <p>This pointer pattern extracts book</p>
+    </cRefPattern>
+   </refsDecl>
+  </encodingDesc>
+<profileDesc>
+<langUsage>
+<language ident="grc">Greek</language>
+<language ident="lat">Latin</language>
+</langUsage>
+</profileDesc>
```

Comment:
missing the change log where you would indicate the EpiDoc and CTS updating

Reply:
for example
`  <revisionDesc>
       <change who="Lisa Cerrato" when="2023-04-10">EpiDoc and CTS conversion and other cleanup</change>
    </revisionDesc>`